docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -215,54 +213,110 @@ for more information about the environment variables.
 
 The index name to write events to. The default is
 +"{beatname_lc}-%\{[beat.version]\}-%\{+yyyy.MM.dd\}"+ (for example,
-+"{beatname_lc}-{version}-2017.04.26"+). If you change this setting, you also
++"{beatname_lc}-{version}-{localdate}"+). If you change this setting, you also
 need to configure the `setup.template.name` and `setup.template.pattern` options
-(see <<configuration-template>>).
-
-When loading the pre-built Kibana dashboards via {beatname_uc},
-you also need to set the `setup.dashboards.index` option (see <<configuration-dashboards>>).
+(see <<configuration-template>>). If you are using the pre-built Kibana
+dashboards, you also need to set the `setup.dashboards.index` option (see
+<<configuration-dashboards>>).
 
+You can set the index dynamically by using a format string to access any event
+field. For example, this configuration uses a custom field, `fields.log_type`,
+to set the index: 
 
-===== `indices`
+["source","yaml",subs="attributes"]
+------------------------------------------------------------------------------
+output.elasticsearch:
+  hosts: ["http://localhost:9200"]
+  index: "%\{[fields.log_type]\}-%\{[beat.version]\}-%\{+yyyy.MM.dd}\" <1>
+------------------------------------------------------------------------------
```

Comment:
Hm, we did have some note around changing APM indices, that are gone with this update. (Either they were never in beats, or they have been overwritten by some beats change).

Reply:
I was talking about this note https://github.com/elastic/apm-server/pull/1527/files/d22b30b7fb72148fbc8ec994b296d0c7c989a7ff#diff-fde09970178d77481d2fcead190567caL243.
It could be though that those changes were only made here and not in libbeat, I recall some conflict with updates in beats.